chat-components/src/main/java/com/pubnub/components/chat/provider/ChatProvider.kt

```diff
@@ -117,6 +120,12 @@ fun ChatProvider(
         LocalUser provides pubNub.configuration.userId.value,
         LocalChannel provides channel,
 
+        // RTL support by locale
```

Comment:
@kazydek it's quite important. It forces application to use the system layout direction (left to right or right to left), depends on user locale